### DISCUSSION TITLE
Using as frontend only

### DISCUSSION THREAD

**Discussion by barsdeveloper** [DioxusLabs/dioxus#1004] [Q&A] [answered]
Hello, I'm doing some research to replace JS with Rust for a project I'm working on. I was wondering, can this framework be used in a frontend only context? If so, what is the total dependency size the browser has to download for a hello world application?
Thanks for the patience.

**Comment by ealmloff** [comment, marked as answer]
👋 Yes, Dioxus can be to replace just the frontend of a project. A [hello world example is ~62 KB GZIPed](https://github.com/DioxusLabs/dioxus/issues/732)
